packages/scandipwa/src/component/MyAccountOrderTotals/MyAccountOrderTotals.component.js

```diff
@@ -54,7 +54,11 @@ export class MyAccountOrderTotals extends PureComponent {
     }
 
     renderDiscount({ label, amount: { value } }, index) {
-        return this.renderPriceLine(__('Discount (%s)', label), value, null, {}, `discount-${index}`);
+        if (label === 'Discount') {
+            return this.renderPriceLine(__('Discount'), -value, null, {}, `discount-${index}`);
+        }
+
+        return this.renderPriceLine(__('Discount (%s)', label), -value, null, {}, `discount-${index}`);
```

Comment:
Better to make check if it is default discount without label for it, this case if no label -> render default label `Discount`